spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
Yep, I think this wont do, and we will need to add a tuple or something to the shortcuts list and fix where needed in `config.py`?

```python
...
            ('shortcuts',
             {
              # ---- Global ----
              # -- In spyder.py
              '_/close pane'): ("Shift+Ctrl+F4", _('Close pane')),
              '_/preferences'): ("Ctrl+Alt+Shift+P", _('Preferences')),
...
```